### DISCUSSION TITLE
Opening help file in web browser?

### DISCUSSION THREAD

**Discussion by PMassicotte** [REditorSupport/vscode-R#1786] [Q&A]
I am using this key binding to open help:

```json
  {
    "key": "ctrl+f1",
    "command": "r.helpPanel.openForSelection",
    "when": "editorTextFocus && editorLangId == 'r' || editorTextFocus && editorLangId == 'rmd' || editorTextFocus && editorLangId == 'quarto'"
  },
```

Using ctr + f1 when the cursor is over a function, this open the help file in a new panel inside VSCode. Is it possible to open it in a web browser? This is working if I type `?plot` in the console.

So I guess the question is more, can I replace `"command": "r.helpPanel.openForSelection"` with something else so the help page  open in my browser?

Thank you in advance

[a comment by an automated account is not shown]

**Comment by ThomasSoeiro** [comment]
> So I guess the question is more, can I replace "command": "r.helpPanel.openForSelection" with something else so the help page open in my browser?

@eitsupi @ManuelHentschel Any hint?
